salon_creator/lib/app.dart

```diff
@@ -32,6 +33,7 @@ class SalonCreatorApp extends StatelessWidget {
         '/terms': (context) => TermsScreen(),
         '/user/profile/get': (context) => UserProfileScreen(),
         '/user/profile/update': (context) => UserProfileEditScreen(),
+        'lesson_creation': (context) => LessonCreationScreen(),
```

Comment:
```suggestion
        '/lesson/create': (context) => LessonCreationScreen(),
```

This is more REST-like